### PR TITLE
Add remaining series to conversion and apply prefixes / suffixes

### PR DESCRIPTION
**Pull request by mico**
related #1

### CODE REVIEW

File: nist_pubid.gemspec [hunk 1]
```diff
@@ -24,6 +24,8 @@ Gem::Specification.new do |spec|
   spec.require_paths = ['lib']
   spec.required_ruby_version = Gem::Requirement.new('>= 2.5.0')
 
+  spec.add_runtime_dependency "asciidoctor"
```
Comment (ronaldtse) [inline nist_pubid.gemspec:27]: Why do we need asciidoctor?

File: nist_pubid.gemspec [hunk 2]
```diff
@@ -27,4 +27,7 @@ Gem::Specification.new do |spec|
   spec.add_development_dependency 'pry-byebug'
   spec.add_development_dependency 'rake', '~> 13.0'
   spec.add_development_dependency 'rspec', '~> 3.0'
+  spec.add_development_dependency "rubocop", "~> 1.23.0"
+  spec.add_development_dependency "rubocop-rails", "~> 2.12.4"
```
Comment (ronaldtse) [inline nist_pubid.gemspec:31]: Do we really need rubocop-rails?

Comment (mico) [reply nist_pubid.gemspec:31]: Because of dependency here https://github.com/riboseinc/oss-guides/blob/master/ci/rubocop.yml#L19-L20

**Comment by mico** [comment]
@ronaldtse almost done. could you give me some feedback on it? and I'll do final refactoring

File: nist_pubid.gemspec (hunk 2 above)
Comment (ronaldtse) [reply nist_pubid.gemspec:31]: Can you remove these two dependencies from this gem? It is really unnecessary. Thanks.

**Review by ronaldtse — CHANGES_REQUESTED** [review]

**Comment by ronaldtse** [comment]
We would want to have all the tests imported from the README but that could be a separate PR.

File: nist_pubid.gemspec (hunk 2 above)
Comment (mico) [reply nist_pubid.gemspec:31]: Are you sure? How to deal with that?
```
% bundle exec rubocop
cannot load such file -- rubocop-rails
```

**Comment by mico** [comment]
@ronaldtse I believe it's ready, except working with DOI format and "Section, Supplement, Index, Insert, Errata".

**Review by ronaldtse — APPROVED** [review]

**Pull request merged**